docs/src/pages/guides/FHIRValidationGuide.md

```diff
@@ -224,3 +224,124 @@ The IBM FHIR Server provides a basic implementation of the $validate operation t
 ## ValueSet membership checking (FHIRPath `memberOf` function)
 
 Coded elements (code, Coding, CodeableConcept data types), maybe have a binding element that specifies a ValueSet that that element is bound to. This means that the coded element must have a value that comes from that value set. The FHIR profile component will expand value sets according to the ValueSet [expansion algorithm](http://hl7.org/fhir/valueset.html#expansion) for ValueSets that include CodeSystem resources available via the FHIR registry component.
+
+# fhir-path-cli - command line execution of fhirpath over a FHIR Resource
+
+The fhir-path-cli enables commandline processing of a FHIR Path over a FHIR resource and outputs the results. 
+
+The options for the command line utility are:
+- `--path 'fhir-path'`
+- `--format ['json'|'xml']` Default is 'json'
+- `--type ['stdin'|'file'|'string']`
```

Comment:
can `type` have a default too?  maybe `stdin` or `file`...whichever we think will be most common

Reply:
actually, maybe we can just remove this and deduce it from the other arguments?
* `stdin` can be the default
* if `--file` is present then its type=file
* if `--resource` is present then its type=string

Reply:
I thought about that too. I decided to adhere to what was in the issue. 
Maybe we infer if it doesn't exist?

Reply:
I fine with that.  Although I can't imagine anyone wanting to specify it when they don't have to!

Reply:
Fixed... interpolating the type from the command parameters.